spec/models/bookmark_query_spec.rb

```diff
@@ -5,7 +5,22 @@
   it "should allow you to perform a simple search" do
```

Comment:
As we have quite a lot of search-related specs now, I'm wondering if we should put them into the `search` subfolder so they're not lost in the pile in `models`. But not something that urgently needs to be done right now.